server/graphql/v2/object/ContributionStats.js

```diff
@@ -0,0 +1,12 @@
+import { GraphQLFloat, GraphQLInt, GraphQLObjectType } from 'graphql';
+
+export const ContributionStats = new GraphQLObjectType({
+  name: 'ContributionStats',
+  description: 'Contribution statistics related to the given accounts',
+  fields: () => ({
+    numContributions: { type: GraphQLInt, description: 'The total number of contributions' },
```

Comment:
I personally find the `_Count` pattern clear than `num_` (eg. `numContributions` -> `contributionsCount`). And it's consistent with what we do on `Collections`, where we say `totalCount`.

@znarf any opinion?

Reply:
Absolutely.